index.js

```diff
@@ -2441,6 +2441,24 @@
         return n < 0 || negativeZero(n) ? Nothing() : slice(0, -n, xs);
       });
 
+  //# reverse :: [a] -> [a]
+  //.
+  //. Takes an array and returns an array of the its elements in
+  //. reversed order.
```

Comment:
Let's make it clear that this function is not Array-specific. I suggest:

_Returns the elements of the given [list](https://github.com/sanctuary-js/sanctuary#list-pseudotype) in reverse order._